docs/btcli-permissions.md

```diff
@@ -25,19 +25,19 @@ Interacting with Bittensor generally falls into one of three levels of security,
 
 The workstations you use to do this work can be referred to as a permissionless workstation (requiring neither private key), a coldkey workstation or a hotkey workstation, depending on which private key is provisioned.
 
-1. A **permissionless workstation** has only coldkey *public keys* on it. which are sufficient for viewing all public information about a wallet, such as TAO and alpha stake balances. Information about subnets, miners, and validators can be viewed without introducing the security risk of initializing your private keys on a device.
+1. A **permissionless workstation** has only coldkey *public keys* on it. These are sufficient for viewing all public information about a wallet, such as TAO and alpha stake balances. That way, information about subnets, miners, and validators can be viewed without introducing the security risk of initializing your private keys on a device.
 
     :::tip coldkey workstation security
     See [Permissionless workstation](./getting-started/coldkey-hotkey-security#permissionless-workstation)
     :::
 
-1. A **coldkey workstation** contains coldkey in the `wallet_path`. For any coldkey associated with mainnet TAO, the coldkey workstation should be held to the highest possible security standards. 
+1. A **coldkey workstation** contains a coldkey in the `wallet_path`. For any coldkey associated with mainnet TAO, the coldkey workstation should be held to the highest possible security standards. 
 
     :::tip coldkey workstation security
     See [Coldkey workstation](./  getting-started/coldkey-hotkey-security#coldkey-workstation)
     :::
 
-1. **A hotkey workstation**, which is generally a servers used for mining or validation, contains a hotkey private key in the `wallet_path` in the `btcli config`, as well as a coldkey public key for the corresponding coldkey. Compromised hotkeys can damage your reputation if they are used to maliciously submit false weights (if you are a validator), or bad work as a miner. However, ownership of TAO or alpha stake can only be transferred with a coldkey, and a leaked hotkey can be swapped using the coldkey; therefore hotkey leaks are far less dangerous than coldkey leaks.
+1. **A hotkey workstation**—which is generally a server used for mining or validation—contains a hotkey private key in the `wallet_path` located in the `btcli config`, as well as a coldkey public key for the corresponding coldkey. Compromised hotkeys can damage your reputation if they are used to maliciously to submit false weights as a validator, or bad work as a miner. However, ownership of TAO or alpha stake can only be transferred with a coldkey, and a leaked hotkey can be swapped using the coldkey; therefore hotkey leaks are far less dangerous than coldkey leaks.
```

Comment:
```suggestion
1. **A hotkey workstation**&mdash;generally mining or validation server&mdash;contains a hotkey private key in the `wallet_path` located in the `btcli config`, as well as a coldkey public key for the corresponding coldkey. Compromised hotkeys can damage your reputation if they are used to maliciously to submit inaccurate weights as a validator, or bad work as a miner. However, ownership of TAO or alpha stake can only be transferred with a coldkey, and a leaked hotkey can be swapped using the coldkey; therefore hotkey leaks are far less dangerous than coldkey leaks.
```

Reply:
You can make a renderable em-dash with shift option dash. It rendered as a full emdash on my local build FWIW. You just can't tell in git. Would you prefer I use $mdash so that it is obvious in git?